draft-wussler-openpgp-pqc.md

```diff
@@ -830,69 +754,70 @@ The operation `ecdhKem.decap()` is defined as follows:
     chosen according to {{tab-ecdh-nist-artifacts}} or
     {{tab-ecdh-brainpool-artifacts}}
 
-### Kyber-KEM {#kyber-kem}
+### ML-KEM {#mlkem-ops}
 
-Kyber-KEM features the following operations:
+ML-KEM features the following operations:
 
-    (kyberCipherText, kyberKeyShare) <- kyberKem.encap(kyberPublicKey)
+    (mlkemCipherText, mlkemKeyShare) <- ML-KEM.Encaps(mlkemEncapsKey)
 
 and
 
-    (kyberKeyShare) <- kyberKem.decap(kyberCipherText, kyberPrivateKey)
+    (mlkemKeyShare) <- ML-KEM.Decaps(mlkemCipherText, mlkemDecapsKey)
 
-The above are the operations Kyber.CCAKEM.Enc() and Kyber.CCAKEM.Dec() defined
-in [Kyber-Subm].
+The above are the operations `ML-KEM.Encaps` and `ML-KEM.Decaps` defined in
+[FIPS-203]. Note that `mlkemEncapsKey` is public and `mlkemDecapsKey` is
+private keying material.
 
-Kyber-KEM has the parameterization with the corresponding artifact lengths in
-octets as given in {{tab-kyber-artifacts}}. All artifacts are encoded as
-defined in [Kyber-Subm].
+ML-KEM has the parameterization with the corresponding artifact lengths in
+octets as given in {{tab-mlkem-artifacts}}. All artifacts are encoded as
+defined in [FIPS-203].
 
-{: title="Kyber-KEM parameters artifact lengths in octets" #tab-kyber-artifacts}
-Algorithm ID reference | Kyber-KEM    | Public key | Secret key | Ciphertext | Key share
-----------------------:| ------------ | ---------- | ---------- | ---------- | ---------
-29, 31, 33             | kyberKem768  | 1184       | 2400       | 1088       | 32
-30, 32, 34             | kyberKem1024 | 1568       | 3186       | 1568       | 32
+{: title="ML-KEM parameters artifact lengths in octets" #tab-mlkem-artifacts}
+Algorithm ID reference | ML-KEM                    | Public key | Secret key | Ciphertext | Key share
+----------------------:| ------------------------- | ---------- | ---------- | ---------- | ---------
+29, 31, 33             | ML&#x2011;KEM&#x2011;768  | 1184       | 2400       | 1088       | 32
+30, 32, 34             | ML&#x2011;KEM&#x2011;1024 | 1568       | 3168       | 1568       | 32
```

Comment:
```suggestion
Algorithm ID reference | ML-KEM      | Public key | Secret key | Ciphertext | Key share
----------------------:| ----------- | ---------- | ---------- | ---------- | ---------
29, 31, 33             | ML-KEM-768  | 1184       | 2400       | 1088       | 32
30, 32, 34             | ML-KEM-1024 | 1568       | 3168       | 1568       | 32
```

Reply:
See https://github.com/openpgp-pqc/draft-openpgp-pqc/pull/60#discussion_r1311121166

Reply:
Let it break as it break, no special character encodings.